tests/data/test_json.c

```diff
@@ -694,6 +699,23 @@ test_parse_string(void **state)
     assert_ptr_equal(st->dt, NULL);
 }
 
+static void
+test_parse_ll_empty(void **state)
+{
+    struct state *st;
+    const char *modules[] = {"empty_leaf_list"};
```

Comment:
Here you must also put the module name, there is no module with this name.

Reply:
Apologies, I renamed the files and missed the exact name.